app/jobs/process_pull_request_update_job.rb

```diff
@@ -1,7 +1,51 @@
 class ProcessPullRequestUpdateJob < ApplicationJob
+  include Mandate
```

Comment:
```suggestion
  extend Mandate::Memoize
```

Unless we're using the full `call` flow, I try to include only the bits I need, so as not to clash with other stuff accidentally.